students/DennisLee/lesson04/dict_lab.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+
+def dict1():
+    d = {'name': 'Chris', 'city': 'Seattle', 'cake': 'Chocolate'}
+    print(d)
+    del d['cake']
+    print(d)
+    d['fruit'] = 'Mango'
+    print(d.keys())
+    print(d.values())
+    print('cake' in d.keys())
```

Comment:
The `in` operator test for key membership in dicts so this can be `print('cake' in d)`